perfkitbenchmarker/linux_benchmarks/cloudsuite_data_serving_benchmark.py

```diff
@@ -212,10 +210,6 @@ def CleanupServer(vm):
     vm.RemoteCommand('sudo docker stop cassandra-server')
     CleanupServerCommon(vm)
 
-  def CleanupClient(vm):
-    vm.RemoteCommand('sudo docker rmi cloudsuite/data-serving:client')
```

Comment:
Is this not needed because we assume the images will be removed when docker is uninstalled?